### PR TITLE
added skeleton file for LA

### PR DESCRIPTION
**Pull request by SnowCatSci**
This file will be more things to do in greater LA area.

### CODE REVIEW

File: bruin2015.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Add things to LA file
+
+### Things to do in LA
+
+[] take the ferry to Catalina Island
```
Comment (d-hawkins) [inline bruin2015.md:5]: Cool, I haven't had a chance to do that. Have you gone diving over there? I've seen cool photos of the kelp forests.

**Review by i3ince — CHANGES_REQUESTED** [review]
Add sporting events

File: bruin2015.md (hunk 1 above)
Comment (SnowCatSci) [reply bruin2015.md:5]: Haven't gone diving, not certified, but have also heard there is great diving! Snorkeling is also really fun, so much to see! I've been camping on the island every summer, great relaxation and island time!

**Review by i3ince — APPROVED** [review]

**Comment by SnowCatSci** [comment]
@i3ince I added some suggestions. Feel free to make further ones.

File: bruin2015.md (hunk 1 above)
Comment (d-hawkins) [reply bruin2015.md:5]: I'll have to go out there sometime!

**Pull request merged**